test/jest/fixtures.js

```diff
@@ -131,12 +137,74 @@ module.exports = function registerFixtures(global) {
     await test(playwright.toImpl);
   });
 
+  const allBrowsers = new Set();
```

Comment:
nit: allBrowsers -> closables? Or type it so that we can see there are servers and contexts in it.

Reply:
Did both.